### ISSUE TITLE
New feature of windows splited

### ISSUE THREAD

**Issue by sunjar2020** [highfestiva/finplot#308]
Hi, can you consider adding a new feature to finplot? That is to divide the plotting window into four sub-windows, so that the plottings of different periods can be displayed in their respective windows at the same time, such as 5 minutes, 15 minutes, 30 minutes and so on.

Thank you.

**Comment by highfestiva** [comment]
No, but you could implement that yourself outside of the library. It's pretty easy to do, just plot 5m, resample to 15m and plot on second axis, etc.

**Comment by highfestiva** [comment]
You can embed the four axes yourself. See embed.py example for more info.

**Comment by sunjar2020** [comment]
Thanks, I'm going to spend some time study this example.
